pkg/nuget/parse.go

```diff
@@ -0,0 +1,84 @@
+package nuget
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"io"
+)
+
+// The lockfile that nuget uses (packages.lock.json) works basically like this:
+// Each 'build type' (for example .NETCoreApp,Version=v3.1) have a set of dependencies.
+// Those dependencies can be 'Direct', 'Transitive' and 'Project'
+// (meaning of those are quite self explanatory I think). The dependency consists of a string
+// value key (package name) and an object, from which we need to fetch a few different values.
+// The following values should be parsed right away: resolved (resolved version),
+// type (this information might be worth having) and finally, it's dependencies.
+// The dependencies in this stage of the file are only a string map where the key is package name
+// and value is package version.
+
+type LockFile struct {
+	Version      int
+	Dependencies map[string]map[string]Dependency
+}
+
```

Comment:
We probably should call it `Targets` according to the source code.
https://github.com/NuGet/NuGet.Client/blob/32cbb0ad13cf4b1abdb961b7ac111463e5f7b31a/src/NuGet.Core/NuGet.ProjectModel/ProjectLockFile/PackagesLockFileFormat.cs#L95

```suggestion
type LockFile struct {
	Version      int
	Targets map[string]Dependencies `json:"dependencies"`
}

type Dependencies map[string]Dependency
```